File: C#JAVA_Client_Server/ClientServcerC#Protocols/Solution1/Domain/Sofer.cs
using System;

namespace Domain
{
    [Serializable]
    public class Sofer : IObject<int>
    {
        String numeSofer;
        private int idEchipa;
        private int idMasina;

        public string NumeSofer
        {
            get => numeSofer;
            set => numeSofer = value;
        }

        public int IdEchipa
        {
            get => idEchipa;
            set => idEchipa = value;
        }

        public int IdMasina
        {
            get => idMasina;
            set => idMasina = value;
        }

        public Sofer(string numeSofer, int idEchipa, int idMasina)
        {
            this.numeSofer = numeSofer;
            this.idEchipa = idEchipa;
            this.idMasina = idMasina;
        }

        public override string ToString()
        {
            return base.ToString()+" "+numeSofer+" "+idEchipa.ToString()+" "+idMasina.ToString();
        }

        public Sofer()
        {
            
        }

        public int Id { get; set; }
    }
}

File: C#JAVA_Client_Server/ClientServcerC#Protocols/LongExample/Domain/src/main/java/entities/domain/Sofer.java
package entities.domain;

public class Sofer extends Entitate<Integer> {
    String numeSofer;
    Integer idEchipa;
    Integer idMasina;

    @Override
    public String toString() {
        return "entities.domain.Sofer{" +
                "numeSofer='" + numeSofer + '\'' +
                ", idEchipa=" + idEchipa +
                ", idMasina=" + idMasina +
                '}';
    }

    public String getNumeSofer() {
        return numeSofer;
    }

    public void setNumeSofer(String numeSofer) {
        this.numeSofer = numeSofer;
    }

    public Integer getIdEchipa() {
        return idEchipa;
    }

    public void setIdEchipa(Integer idEchipa) {
        this.idEchipa = idEchipa;
    }

    public Integer getIdMasina() {
        return idMasina;
    }

    public void setIdMasina(Integer idMasina) {
        this.idMasina = idMasina;
    }

    public Sofer(String numeSofer, Integer idEchipa, Integer idMasina) {
        this.numeSofer = numeSofer;
        this.idEchipa = idEchipa;
        this.idMasina = idMasina;
    }
}
